notebooks/image_models/solutions/classification_automl_vision.ipynb

```diff
@@ -0,0 +1,675 @@
+{
```

Comment:
<ul><li>Is it still called AutoML Vision even though it's under Vertex now?</li><li>Should add a learning objective to launch the AutoML training job</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>+1</p>

Reply:
<p>I removed <code>Vision</code> , and added training to learning objective</p>